### PR TITLE
Acceptance tests

### PR DESCRIPTION
**Pull request by kerrr**
Added a basic acceptance test. This particular test is being used to demonstrate the acceptance tests job in the Concourse CI Pipeline.

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -13,10 +13,19 @@
 # it.
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
+
+require 'capybara/rspec'
```
Comment (chrisholmes) [inline spec/spec_helper.rb]: This is adding capybara as a dependency on all of our tests, which could slow them down. We could isolate this behaviour into an `acceptance_helper.rb` that can be required only from the classes that need it.

File: lib/tasks/acceptance_tests.rake [hunk 2]
```diff
@@ -0,0 +1,7 @@
+namespace :acceptance_tests do
+  desc 'Run acceptance tests'
+  task :run, [:url] do |task, args|
+    #Example command: bundle exec rake acceptance_tests:run[verify-self-service-dev.cloudapps.digital]
+    sh "TEST_URL=#{args[:url]} bundle exec rspec spec/system/acceptance --tag acceptance"
```
Comment (chrisholmes) [inline lib/tasks/acceptance_tests.rake]: You can use [RSpec's rake take](https://relishapp.com/rspec/rspec-core/docs/command-line/rake-task) here so you don't have to shell out.

File: spec/system/acceptance/visit_home_controller_acceptance_spec.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
```
Comment (chrisholmes) [inline spec/system/acceptance/visit_home_controller_acceptance_spec.rb]: These tests don't have to be written as system specs or using rails helper as it's only using them for the capybara support. If we introduce an `acceptance_helper.rb` then we can add the capybara setup we need there and remove the rails dependency.

**Comment by kerrr** [comment]
Merging this on another branch.

**Pull request closed without merging**